config/env_spec_request_test.go

```diff
@@ -40,10 +43,10 @@ func TestNilReceivers(t *testing.T) {
 	s.GetAPISpec()
 	s.GetOperation()
 	s.GetParamValue(APIOperationParameter{})
-	s.IsAuthenticated()
-	s.verifyJWTAuthentication("")
+	_ = s.Authenticate()
+	_ = s.verifyJWTAuthentication("")
 	s.getAuthenticationRequirement()
-	s.meetsAuthenticatationRequirements(AuthenticationRequirement{})
+	_ = s.verifyAuthenticatationRequirements(AuthenticationRequirement{})
```

Comment:
nit: this would be a good time to fix this typo (should read: `verifyAuthenticationRequirements`) :joy:

Reply:
haha..thanks for pointing out, Done!